concepticondata/concepticon.tsv

```diff
@@ -3060,3 +3060,58 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
+3072	ENVELOPE	Basic actions and technology	An item to commonly used to package or wrap letters.	Person/Thing	
+3073	GIRAFFE	Animals	A quadruped animal native to Africa, remarkable for the length of its legs and neck.	Person/Thing	
+3074	GUITAR	Speech and language	Musical instrument with usally six strings. 	Person/Thing	
+3075	LEMON	Agriculture and vegetation	Fruit with yellow rind and acidic/sour juice.	Person/Thing	
+3076	LIGHTER	Basic actions and technology	Portable instrument for igniting something.	Person/Thing	
+3077	LIPSTICK	Clothing and grooming	Stick containing wax, oil, or pigments used to apply colour to one's lips.	Person/Thing	
+3078	PAINTBRUSH	Basic actions and technology	Brush used to spread pigments of colour on a surface.	Person/Thing	
+3079	POSTMAN	Modern world	Person who transports and delivers items sent with a postal service.	Person/Thing	
+3080	RACKET	Modern world	Tool used for hitting a ball in tennis-like games.	Person/Thing	
+3081	SEWING MACHINE	Modern world	A machine that partially automates the process of sewing, i.e. uniting pieces of material by using a needle and a thread.	Person/Thing	
+3082	SNOWMAN	Basic actions and technology	A mass of snow sculptured into the shape of a human.	Person/Thing	
+3083	SOFA	The house	Piece of furniture usually stuffed and used for seating two or more persons.	Person/Thing	
+3084	SUNGLASSES	Clothing and grooming	Form of glasses with tinted lenses to protect one's eyes from sunlight.	Person/Thing	
+3085	SWING	Basic actions and technology	Tool used for play with a seat hanging from ropes.	Person/Thing	
+3086	TANK	Warfare and hunting	Armoured vehicle used by the military.	Person/Thing	
+3087	TENNIS BALL	Modern world	Yellow, spherical object used for playing the game of tennis.	Person/Thing	
+3088	THERMOMETER	Basic actions and technology	Instrument used for measuring temperature, mostly by means of a contracting or expanding substanced contained within the instrument. 	Person/Thing	
+3089	UMBRELLA	Clothing and grooming	Portable folding canopy used for protection against weather influences.	Person/Thing	
+3090	WARDROBE	The house	Large object, often made out of wood, used for storing clothes or other fabrics.	Person/Thing	
+3091	WHISTLE (INSTRUMENT)	Basic actions and technology	Wind instrument most often made out of metal or wood used for producing a loud, shrill tone.	Person/Thing	
+3092	BOX (SOMEBODY OR SOMETHING)	Basic actions and technology	The act of using fists to fight or hit.	Action/Process	
+3093	BRUSH (TEETH)	Clothing and grooming	Clean one's teeth by repeatedly grating over them with an object.	Action/Process	
+3094	CLAP	Motion	Make noise by putting one's hand together.	Action/Process	
+3095	CONDUCT	Speech and language	To lead some kind of musical performance.	Action/Process	
+3096	ERUPT	The physical world	Burst of matter from a volcano.	Action/Process	
```

Comment:
sorry, I'm a bit slow today, but ERUPT, is this a verb? What is the German translation in the source?

Reply:
Ausbrechen.